helm/managed/observability/eck/values.yaml

```diff
@@ -52,6 +52,14 @@ eck-stack:
           version: latest
     http:
       service:
+        metadata:
```

Comment:
@GordonSmith these annotations are attached to a Kibana service we manage through this helm chart. 
WsResources will find this service if it's within the same cluster/namespace.